hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestAsyncRegionAdminApi2.java

```diff
@@ -283,4 +285,69 @@ private void splitTest(TableName tableName, int rowCount, boolean isSplitRegion,
     }
     assertEquals(2, count);
   }
+
+  @Test
+  public void testTruncateRegion() throws Exception {
```

Comment:
Could we add another test for for pre check failure for truncate region of meta table?

Reply:
Done. Thanks.